ci/appveyor/test.bat

```diff
@@ -5,17 +5,26 @@
 @rem Required vars:
```

Comment:
This script is still commented out, right?

Reply:
Correct.